src/data.py

```diff
@@ -24,8 +25,8 @@ def __init__(self, parent, xdata, ydata, import_settings = None):
         self.xdata_clipboard = [self.xdata.copy()]
         self.ydata_clipboard = [self.ydata.copy()]
         self.key: str = str(uuid.uuid4())
-        if import_settings is None:
-            import_settings = graphs.get_import_settings(parent) 
+        if(import_settings == None):
```

Comment:
Using` is None`  instead of `== None` is by purpose. The difference is that `== None` checks for value equality (do two objects have the same value. While `is None` checks for reference equality (does our object refer to `None` specifically). As a general rule, you'd like to use `is None` when refering to None.

Unless you've got a good reason to use == here, I'd suggest keep using is, as it's considered best practice.

Reply:
Will change that. At some point I removed this, not considering it breaks stuff and just reimplemented it